_includes/contact_form.html

```diff
@@ -408,8 +408,8 @@ <h3>{{ site.data.[page.lang].settings.contact_page.support_request_form.title }}
     class="usa-form"
   >
     <input type="hidden" name="00NU0000004z90b" id="00NU0000004z90b" value="{{ site.data.language_map.languages[page.lang] | default: 'English' }}" />
-    <input type="hidden" name="00N3R00000AdDdq" id="00N3R00000AdDdq" value="" />
```

Comment:
are these sandbox values? if so, should we override in `_config.yml`?

Reply:
@zachmargolis moved it to config, the syntax reads better that way. we have a number of other hardcoded values on the form.